R/util.R

```diff
@@ -104,6 +110,30 @@ map_dbl_ = function(X, FUN, ...) {
   vapply(X, FUN, FUN.VALUE = numeric(1), ...)
 }
 
+map_lgl_ = function(X, FUN, ...) {
+  vapply(X, FUN, FUN.VALUE = logical(1), ...)
+}
+
+map2_chr_ = function(X, Y, FUN, ...) {
```

Comment:
Since the `...` isn't used (and doesn't really need to be) I'd omit it so that if some future person (probably me 🤷 ) forgets the signature of `map2_chr_()` and does try to pass stuff it will error instead of silently ignoring those args.

```suggestion
map2_chr_ = function(X, Y, FUN) {
```

Reply:
The main reason to keep it is that map2_chr() can pass additional arguments to FUN through ..., rather than needing to use an anonymous function. 

`map2_chr(x, y, sum, na.rm = TRUE)`

Reply:
I'm good either way, up to you